### DISCUSSION TITLE
ACABA SANDIKTAN BANA PUAN ÇIKARMI

### DISCUSSION THREAD

**Discussion by Akrepkapo** [IBAX-io/go-ibax#619] [Soru Cevap - Türkçe]
Evet soru şu acaba bana da kutudan  puan çıkarmı 😂😂@ibaxio @powerpook @BadAssKyle @scottafk @akibax

**Comment by Akrepkapo** [comment]
Bence çıkmaz 😂😂😂

**Reply by Yusufknk** [reply]
Sandıktan tüm üyelere paun çıkacaktır. Şans faktörü devrede olduğu için üyelere göre değişiklik gösterir.
